jjb/3rdparty/boost.yaml

```diff
@@ -0,0 +1,95 @@
+---
+- project:
+    name: boost
+    jobs:
+      - 'boost-build'
+
+    project: ci-management
+    project-name: ci-management
+    build-node: centos7-builder-vfx-2018-2c-1g
+
+    make-opts: '-j8'
+    pre-build: |
+        #!/bin/bash
+        MAJOR_VERSION="1"
+        MINOR_VERSION="61"
+        PATCH_VERSION="0"
+        echo "${{MAJOR_VERSION}}.${{MINOR_VERSION}}.${{PATCH_VERSION}}" > /tmp/artifact_version
+        mkdir $WORKSPACE/target
+        wget https://sourceforge.net/projects/boost/files/boost/1.61.0/boost_1_61_0.tar.bz2/download -O $WORKSPACE/target/boost_1_61_0.tar.bz2
+        cd $WORKSPACE/target && tar xf boost_1_61_0.tar.bz2
+
+    nexus-group-id: io.aswf.3rdparty
+    staging-profile-id: 126694cb53ec54
```

Comment:
This staging ID won't work as it's for opencolorio. I made a new one for 3rdparty and it is "15bbd0a38e296e".